config/ruby.yaml

```diff
@@ -31,10 +31,15 @@ supervisor:
   status_response_timeout: 0.2
   status_update_timeout: 0.2
 
+scripts:
+  activate_alarm: /home/seguro/RSMP/scripts/activate_alarm.sh
```

Comment:
i don't like path to local folders in the repo. preferably. when testing against the ruby tlc, what does this script do?

Reply:
My bad, forgot to remove them before I committed. The script do nothing, I just added them to make sure 'check_scripts' didn't raise any errors when I was testing.